test/integration/patients/worklist/schedule.js

```diff
@@ -582,10 +582,12 @@ context('schedule page', function() {
   specify('reduced patient schedule employee', function() {
```

Comment:
I updated this test to load three actions into the reduced schedule list:

1. One for linking to a patient action.
2. One for linking to a flow action.
3. One that does not have a due date.

Using those three actions, I added the following to the test:

* Verify the search filter works.
* Verify that the list only includes two items. The patient action and flow action should show. But the action with no due date should not be included in the list.
* Verify the links for the patient and flow actions work as expected and take the user to the correct pages.